static/src/javascripts/projects/common/modules/experiments/tests/onward-container-names.js

```diff
@@ -0,0 +1,36 @@
+define([
+], function (
+) {
+    return function () {
+        this.id = 'OnwardNames';
+        this.start = '2015-10-20';
+        this.expiry = '2015-11-20';
+        this.author = 'John Duffell';
+        this.description = 'renames the related content container';
+        this.audience = 0.6;
+        this.audienceOffset = 0.4;
+        this.successMeasure = 'The number of onward journeys';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = '';
+        this.idealOutcome = 'the clicks should be significantly higher for one of the buckets';
+
+        this.canRun = function () {
+            return window.guardian.config.page.contentType === 'Article';
+        };
+
+        this.variants = [
+            {
+                id: 'control',
+                test: function () {}
+            },
+            {
+                id: 'test:You might like ...',
```

Comment:
I tried it but it ended up with a hellip in my text... since it's only an AB test I'll leave as-is and I promise to do it properly if it wins ;)  does that sound fair?